api/v1/openapi.yaml

```diff
@@ -3715,6 +3718,23 @@ definitions:
     type: array
     items:
       "$ref": "#/definitions/Port"
+  Workload:
+    description: |-
+      Workloads are Deployment, Statefulset, Daemonset, ReplicationController, 
+      CronJob, Job, DeploymentConfig (OpenShift), etc).
+
+      +deepequal-gen=true
+      +k8s:deepcopy-gen=true
+    properties:
+      name:
+        description: workload's name
+        type: string
+      namespace:
+        description: workload's namespace
+        type: string
+      kind:
```

Comment:
suggestion: a bare `kind` is insufficient; this should also have a `group` field.

Reply:
hubble's workload struct is 
https://github.com/cilium/cilium/blob/a0899b9aeb6f14fe438c76fd49396b51fb921733/api/v1/flow/flow.pb.go#L2070
```
type Workload struct {
    state         protoimpl.MessageState
    sizeCache     protoimpl.SizeCache
    unknownFields protoimpl.UnknownFields
    Name          string `protobuf:"bytes,1,opt,name=name,proto3" json:"name,omitempty"`
    Kind          string `protobuf:"bytes,2,opt,name=kind,proto3" json:"kind,omitempty"`
}
```
Since `Group` filed originally is not added here, I didn't consider `Group` filed there either.
So do we need to change this api to add `Group` file？

Reply:
Aha, I didn't realize that. Hmm. Bummer. I wonder if any of the hubble contributors can chime in here.

Reply:
I don't think users will use `group` in the metrics very much to be honest, so I think it's fine. That said, it might still be a good idea to add the `Group` into the endpoint.

Reply:
I'll try to do work on this later. Maybe commit in another new pr ?